buildbot/configure.py

```diff
@@ -103,10 +94,28 @@ def do_configure(args):
         llvm_build_shared_libs = 'ON'
 
     if args.use_lld:
-      llvm_enable_lld = 'ON'
+        llvm_enable_lld = 'ON'
 
-    if build_libclc:
-        llvm_enable_projects += ';libclc'
+    # CI Default conditionally appends to options, keep it at the bottom of
+    # args handling
```

Comment:
@alexbatashev, @vladimirlaz, FYI.
When we originally discussed `--ci-defaults` option, I thought that these defaults can be overridden by additional options. E.g. `--ci-defaults --option-X=val`, so that if `--ci-defaults` sets `--option-X`, users can replace the default value.
Unfortunately, I don't see this requirement to be documented in the code and it seems to be not true for options modified inside this `if-statement`.

Reply:
@bader it’s a good point, we shall fix it in the future